modules/rhel-compute-requirements.adoc

```diff
@@ -8,36 +8,39 @@
 [id="rhel-compute-requirements_{context}"]
 = System requirements for RHEL compute nodes
 
-The Red Hat Enterprise Linux (RHEL) compute machine hosts, which are also known as worker machine hosts, in your {product-title} environment must meet the following minimum hardware specifications and system-level requirements.
+The Red Hat Enterprise Linux (RHEL) compute, or worker, machine hosts in your {product-title} environment must meet the following minimum hardware specifications and system-level requirements:
 
 * You must have an active {product-title} subscription on your Red Hat account. If you do not, contact your sales representative for more information.
 
-* Production environments must provide compute machines to support your expected workloads. As a cluster administrator, you must calculate the expected workload and add about 10 percent for overhead. For production environments, allocate enough resources so that a node host failure does not affect your maximum capacity.
+* Production environments must provide compute machines to support your expected workloads. As a cluster administrator, you must calculate the expected workload and add about 10% for overhead. For production environments, allocate enough resources so that a node host failure does not affect your maximum capacity.
 * Each system must meet the following hardware requirements:
 ** Physical or virtual system, or an instance running on a public or private IaaS.
 ifdef::openshift-origin[]
 ** Base OS: CentOS 7.4.
 endif::[]
 ifdef::openshift-enterprise,openshift-webscale[]
-** Base OS: link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html-single/installation_guide/index[RHEL 7.8 - 7.9] with "Minimal" installation option.
+** Base OS: link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html-single/installation_guide/index[RHEL 7.9] or link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/performing_an_advanced_rhel_installation/index[RHEL 8] with "Minimal" installation option.
```

Comment:
RHEL docs arch vary slightly between 7 & 8, so for 8, I've linked to "Performing an advanced RHEL installation" since the "Installation Guide" is no longer a link AFAICT.

Reply:
RHEL 8 Server support (non-RHCOS) won't be in OCP 4.8. Only support for RHEL 7.9 Server and RHCOS.